doc/source/api-reference/qibolab.rst

```diff
@@ -105,6 +105,28 @@ SWIPHT
    :members:
    :member-order: bysource
 
+Data structures and sweeping
+============================
+
+.. TODO: Add documentation
+
+.. autoclass:: qibolab.platforms.abstract.Qubit
+   :members:
+   :member-order: bysource
+
+.. autoclass:: qibolab.result.ExecutionResults
+   :members:
+   :member-order: bysource
+
+
+.. autoclass:: qibolab.result.AveragedResults
+   :members:
+   :member-order: bysource
+
+.. autoclass:: qibolab.sweeper.Sweeper
+   :members:
+   :member-order: bysource
+
```

Comment:
I expect this to be autogenerated, so in principle it could be gitgnored.
At the moment we're a bit in a rush, but we should keep an eye on these things not to accumulate technical debt.

Reply:
Actually I put this lines directly in the doc. I just wanted to have something in the documentation regarding the new classes that we introduced during the refactor. 

> At the moment we're a bit in a rush, but we should keep an eye on these things not to accumulate technical debt.

I agree. Before the release you should go through the documentation to fix this type of issues. As you said we can do this in a separate PR.